src/DataLayer/utils/AfmUtils.ts

```diff
@@ -321,9 +323,14 @@ export function appendFilters(
         dateFilters.push(afmDateFilter);
     }
 
-    const afmAttributeFilters = afmFilters.filter(filter => !AFM.isDateFilter(filter));
+    const afmNonDateFilters = afmFilters.filter(filter => !AFM.isDateFilter(filter));
 
-    const filters = compact([...afmAttributeFilters, ...attributeFilters, ...dateFilters]);
+    const filters = compact([
+        ...afmNonDateFilters,
+        ...attributeFilters,
+        ...dateFilters,
+        ...(measureValueFilters || []),
```

Comment:
Isn't the `compact` function handling the undefined value so the `|| []` is not necessary?

Reply:
It is, otherwise spread operator on possibly undefined value would throw an error.